build/vss.js

```diff
@@ -70,6 +70,7 @@ async function vssEverything () {
         'package.json',
         'LICENSE.txt',
         'keys.json',
+        'skip-tests.json',
```

Comment:
why do we need to copy this file to the python folder?

Reply:
the same reason we copy another file(s) above it - keys.json and etc.
if we shouldn't copy it, then we shouldn't copy those above ones too.
not?

Reply:
I don't think this is needed tbh because the python package does not need these files

Reply:
the thing is that why do we copy `keys.json`? the same purpose is for `skiped-tests.json` because we have just moved some parts from `keys.json` to reside there, thus we have to include both files.

i.e. here https://github.com/ccxt/ccxt/pull/17075/files#diff-f79e089917fbf457751403ee8c052cfcf73e8e6d1dad6ce628bdee446ec83e43R185

that rootpath resolves to python root path, from where we include keys.json.

can you tell in details , wdyt?

Reply:
@ttodua that link is not pointing to any specific line so I'm not sure what you're referring to, but I don't see why we can't use a relative path to access the file in the root dir `ccxt/`, but ok this is not critical let's keep it as is for now

Reply:
maybe python folder needs to be isolated folder, with its own required json files, so, as long as we copy keys.json, we should do that for skip-json too, as they both have combined purpose for testing.
I think we can amend it later if we see that is not needed.